### ISSUE TITLE
Piecash is dead

### ISSUE THREAD

**Issue by joshuabach** [joshuabach/gnucash-web#61]
See sdementen/piecash#215. They are looking for a new maintainer.

GnuCash Web cannot function without piecash.

**Comment by ducpham01** [comment]
That's sad. I just found this awesome project but now... Do you keep developing this GnuCash Web despite the dying of piecash?

**Comment by joshuabach** [comment]
Yes, I will keep working on GnuCash web. The limiting factor in this regard is my own time, not not dependency status :-)

**Comment by tctlrd** [comment]
## [Piecash](sdementen/piecash) is being reinvigorated!
[Piecash](sdementen/piecash) is being reinvigorated under the new name "[piecash2](/sdementen/piecash2)", see piecash issue [#231](/sdementen/piecash/issues/231).

All efforts towards [piecash2](/sdementen/piecash2) are loudly **applauded**!
